src/xsarsea/windspeed/utils.py

```diff
@@ -1,10 +1,15 @@
 import os
+from pathlib import Path
```

Comment:
curiosity question: why `pathlib` rather than `os` ? Is there a benefit?

Reply:
no i guess its the same, i changed it